frontend/src/views/Bridge.vue

```diff
@@ -165,6 +190,20 @@
         <div v-else-if="nftType === 'character'">
           <h3 class="text-center p-4">{{$t('bridge.noCharactersStored')}}</h3>
         </div>
+        <div v-if="nftType === 'shield' && storedNftsIds.length !== 0">
+          <nft-list
+          v-model="selectedNftId"
+          :characterIds="[storedNftsIds]"
```

Comment:
Can you please check again if we need all of these attributes? We're showing shields here, but we're including :characterIds and we are using nftDysplay flag, is this needed?